plugin-core/src/main/scala/com/alpine/plugin/core/OperatorParameters.scala

```diff
@@ -89,4 +89,5 @@ trait OperatorParameters {
    */
   def getParameterIds: Iterator[String]
 
+  def getAdvancedSparkParameters: scala.collection.mutable.Map[String, String]
```

Comment:
That's fine. I'll merge it as is for this alpha version, and we can finalise later.